### ISSUE TITLE
3.68 doesn't work

### ISSUE THREAD

**Issue by operammm** [odebroqueville/contextSearch#120]
firefox 60.0.2 (x64) win10 (x64)
If i closed icon showing and restart browser contextSearch is not displayed in the right-click menu

**Comment by odebroqueville** [comment]
Hi @operammm , I'm aware of that... it's a work in progress. I mentioned it wasn't working in the commit. I would suggest you stick with the releases on AMO. Thkx

**Comment by odebroqueville** [comment]
@operammm What do you mean by "If i closed icon showing"?

**Comment by operammm** [comment]
Means to turn off the display of search engine icons in the context menu list and search engine icons via the 'besticon' API

**Comment by odebroqueville** [comment]
You're absolutely right @operammm. I was able to reproduce this bug. In the mean time, I would recommend not re-starting your browser. The problem is also easily repaired if you check the 'fetch favicons' option and uncheck it again if you don't want to display them.
